DOCS-1222: Update PCA9685 I2C config info (#2015)
viamrobotics/docs: docs/components/board/pca9685.md

{
  "board_name": "<your-board-name>",
  "i2c_name": "<your-bus-name>",

>   "i2c_bus": <int>,

Author: ```suggestion
  "i2c_bus": “<bus-identifier>”,
```

```json {class="line-numbers linkable-line-numbers"}
{

>   "i2c_bus": 1

Author: ```suggestion
  "i2c_bus": “1”
```

      "type": "board",
      "namespace": "rdk",
      "attributes": {
        "board_name": "<your-board-name>",
        "i2c_name": "<your-bus-name>",

>         "i2c_bus": <int>,

Author: ```suggestion
        "i2c_bus": “<string>”,
```

<!-- prettier-ignore -->
| Name | Type | Inclusion | Description |
| ---- | ---- | --------- | ----------- |
| `board_name` | string | Optional | The `name` of the board with GPIO pins your `pca9685` is [connected to](https://learn.adafruit.com/16-channel-pwm-servo-driver/hooking-it-up). |

> | `i2c_name` | string | Optional | The `name` of the `i2c` your `pca9685` is connected to, as configured in your main board's section of the config file. |

Reviewer: ```suggestion
| `i2c_name` | string | Optional | The `name` of the `i2c` your `pca9685` is connected to, as configured in your main board's JSON config. |
```
(opt) another way to phrase it that might be simpler!

Author: I think if “section” makes sense to you I’ll leave it as-is because it’s all in the same config, just different bits!

> | `i2c_bus` | int | Optional | The index of the board with GPIO pins your `pca9685` is connected to. <br> Example: `1` |

Author: ```suggestion
| `i2c_bus` | string | Optional | The index of the board with GPIO pins your `pca9685` is connected to. Often a number. <br> Example: `”1”` |
```
